dag_files/dag_walkthrough.yml

```diff
@@ -1,28 +1,30 @@
 # Autogenerated steps by walkthrough
```

Comment:
I think we should stop using `dag_walkthrough.yml` and put dag entries in more meaningful places. Maybe `dag_papers.yml` for the current one? Otherwise in the old `dag.yml`.

Reply:
Done! thanks